lib/account.ts

```diff
@@ -2085,6 +2124,184 @@ export async function getIndexInfo(address: string, api: ApiPromise): Promise<Ac
   }
 }
 
+/**
+ * Gets governance deposits (decision and submission) for an address
+ * @param address The address to query
+ * @param api The Polkadot API instance
+ * @returns Array of governance deposits or empty array if not available
+ */
+export async function getGovernanceDeposits(address: string, api: ApiPromise): Promise<GovernanceDeposit[]> {
```

Comment:
This is probably quite a sensitive function... unit testing for this would be amazing...